docs/_posts/2015-03-03-community-roundup-25.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Community Round-up #25"
+layout: post
+author: Matthew Johnston
+---
+
+## React 101
+
+Interest in React has been exploding recently, so it's a good time to explore some great recent tutorials and videos covering getting started. 
+
+Ryan Clarke ([rynclark](https://github.com/rynclark)) provides a [great overview of the basics of React](http://ryanclark.me/getting-started-with-react/) with the goal of building a really simple dropdown nav. 
+
+Formidable Labs ([FormidableLabs](https://github.com/FormidableLabs)) and [Seattle JS](http://www.meetup.com/seattlejs/) recently hosted a series of React, Flux, and Flow workshops, and the first part is available to watch online:
+
+<iframe width="560" height="315" src="//www.youtube.com/embed/[Pd6Ub7Ju2RM](https://www.youtube.com/watch?v=Pd6Ub7Ju2RM)" frameborder="0" allowfullscreen></iframe>
+
+AEFlash ([aearly](https://github.com/aearly)) writeup [some best practices and tips](http://aeflash.com/2015-02/react-tips-and-best-practices.html) to help you avoid potential pitfalls when developing with React. 
+
+Black Mutt Media [take us through their usage of React](http://blackmuttmedia.com/blog/react-tmdb-api/) and Ruby to build an autocomplete field, and some of the pitfalls they encountered along the way.
+
+Our own [sebmarkbage ](https://github.com/sebmarkbage)was on the[ Web Platform Podcast](http://thewebplatform.libsyn.com/31-building-with-reactjs) to have a chat about all aspects of React.
```

Comment:
Sebastian Markbåge

Also, put the spaces outside the links (you do that twice).